integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -401,18 +400,22 @@ public int release(String path, FuseFileInfo fi) {
 
   private int releaseInternal(String path, FuseFileInfo fi) {
     long fd = fi.fh.get();
-    try (LockResource r1 = new LockResource(mFileLocks.get(fd).writeLock())) {
+    try {
       FileInStream is = mOpenFileEntries.remove(fd);
       FileOutStream os = mCreateFileEntries.remove(fd);
       if (is == null && os == null) {
```

Comment:
Hrmmm, should we treat each input and output streams separately/independently? Then, we could just simply do something like
```
if (is == null) {
  return error;
} else {
  synchronized (is) {
    is.close();
  }
}
```

We could even have a helper method which takes a closable.

Reply:
As the previous logic of this method, it return `-ErrorCodes.EBADFD()` only if both is and os are all `null`.

So, I don't know well about how the write the code like your given template, would you please give me some more detail or tips? Thank you very mush!

Reply:
Ok, we don't have to change the existing behavior. I was just wondering if the existing behavior is what we want or not.